cartoframes/viz/__init__.py

```diff
@@ -13,11 +13,14 @@
 from .legend import Legend
 from .widget import Widget
 from .widget_list import WidgetList
-
+from .helpers import __all__ as helpers
+from .widgets import __all__ as widgets
 
 __all__ = [
     'basemaps',
     'themes',
+    'helpers',
```

Comment:
I think this is not needed since they are folders containing and __init__.py file (autodetected).

Also, this may cause the tests to fail.

Reply:
I'll take a look, this is one of the things I'm not 100% sure about sphinx, but I'll test it 👍